src/content/faq/General.md

```diff
@@ -8,20 +8,76 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
+
+
 ## Is GraphQL a database language like SQL?
 
+No, but this is a common misconception.
+
+GraphQL is a specification for how to request and modify data. That data can be provided by [arbitrary functions](/learn/) and those functions can call APIs, databases, [local cache](/learn/caching/) or any representation of data. The main and most common use case is calling APIs.
+
+GraphQL is frequently used for database interactions though. For example, it can be built into a wrapper around a database or a [GraphQL server](https://www.howtographql.com/advanced/1-server/) can query various databases. There are also [services](/code/#services) that implement the GraphQL specification and then allow clients to use GraphQL to query that data source. But it can also be used without a database at all.
+
+For more information on how to get GraphQL to interact with your database, check out our [documentation on resolvers](/learn/execution/#root-fields-resolvers).
+
 ## Does GraphQL replace REST?
 
+No, not necessarily. They both handle APIs and [serve the same purpose](/learn/thinking-in-graphs/#business-logic-layer) from a business perspective. GraphQL is considered an alternative to REST, but it’s not a definitive replacement.
+
+GraphQL and REST can actually co-exist in your stack. For example, you can abstract REST APIs behind a [GraphQL server](https://www.howtographql.com/advanced/1-server/). This can be done by masking your REST endpoint into a GraphQL endpoint using [root resolvers](/learn/execution/#root-fields-resolvers). 
+
+For an opinionated perspective on how GraphQL compares to REST, check out [How To GraphQL](https://www.howtographql.com/basics/1-graphql-is-the-better-rest/).
+
 ## How can I learn GraphQL?
 
+There are many resources available to help you learn GraphQL, including this website. In [our documentation](/learn/), you’ll find a series of articles that explain essential GraphQL concepts and how they work. Our [Community page](/community) is full of resources to reference and groups to join.
+
+For more practical guides, you can visit the [How to GraphQL](https://www.howtographql.com/) fullstack tutorial website. We also have a free online course with edX, [Exploring GraphQL: A Query Language for APIs](https://www.edx.org/course/exploring-graphql-a-query-language-for-apis). 
+
 ## Is GraphQL frontend or backend?
 
+Neither. Instead, it’s [how you can exchange information between the two](https://www.howtographql.com/basics/3-big-picture/). Both the client and the server should understand and communicate through the GraphQL specification.
+
 ## Does GraphQL replace Redux or other state management libraries?
 
+No, GraphQL isn’t a state management library - but it can reduce the need for one.
```

Comment:
I am not sure we should add this very specific JS question to the FAQ. We are not helping here to improve the situation described in the next question.

Reply:
Do you think removing `Redux` would help? This is a _super_ common question, so I believe it should be included in the FAQ eventually - but we could hold it until there are more subsections.

Reply:
I think this is a bit a tough one: I totally get that this is a super common question ... if there would be multiple sections I would recommend to move it.

Maybe we can find a pragmatic solution in the mean time: prefix it with "JS" or so? I just want to reflect a bit more clear that GraphQL is a big ecosystem which has a lot of things to offer outside JS. I think especially for beginners this is important to see and at least get a nudge like "there is more to it than JS". Hope that makes sense.

Reply:
Absolutely makes sense! One of the primary goals for this project is to `Build a resource for everyone, regardless of programming language` so I definitely want to make sure that it aligns. 

I'll remove this question from this PR and keep it in the backlog for once we create subsections within the FAQ 😊